charts/access/jira/values.yaml

```diff
@@ -0,0 +1,72 @@
+# Default values for teleport-plugin-jira.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# replicaCount: 1
+
+image:
+  repository: quay.io/gravitational/teleport-plugin-jira
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+resources: {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+#
+# Plugin specific options
+#
+teleport:
+  address: ""
+  identitySecretName: ""
+  identitySecretPath: "auth_id"
+
+jira:
+  url: ""
+  username: ""
+  apiToken: ""
+  project: ""
+  issueType: "Task"
+
+http:
+  listenAddress: ":8443"
+  publicAddress: ""
```

Comment:
See my earlier remark about which port customers are likely to change.  Users probably won't want to tweak the port between the service and the pod, but instead the port that the service is publicly available on.